tests/tests_suite_test.go

```diff
@@ -32,6 +32,8 @@ import (
 
 	"kubevirt.io/kubevirt/tests"
 	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
+
+	_ "kubevirt.io/kubevirt/tests/network"
```

Comment:
How come this `_` is not needed on line 33?

Reply:
`tests` module is kept in scope. So when you want to use a test helper inside a test, you will need to call `test.MyGreatHelper()` despite the test and `"tests"` helper module live in the same directory.

Reply:
Ohh.. only now I realized you created a new package.
So this import is here to cause this package to be “processed” by ginkgo?

Reply:
The way Ginkgo works is that it has those `Describe` functions called. They are called on import of a package thanks to this construct:

```
var _ = Describe(...
```

All the modules in the current directory do that and thus register their tests to the suite.

With `_` import, we say that we want to import the package only for its side effects. It does not import any content of the referenced package into the current scope though (I hope I understand it correctly).